drivers/network/dd/netkvm/netkvm.inf

```diff
@@ -54,7 +54,7 @@ HKR, Ndi\Interfaces,              LowerRange,          0, "ethernet"
 
 [Parameters]
 HKR, Ndi\Params\ConnectRate,        ParamDesc,  0,          %ConnectRate%
-HKR, Ndi\Params\ConnectRate,        Default,    0,          "1001"
+HKR, Ndi\Params\ConnectRate,        Default,    0,          "10000"
```

Comment:
For what it's worth:
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=%221001%22
```
drivers/network/dd/netkvm/netkvm.inf	
  57 HKR, Ndi\Params\ConnectRate,        Default,    0,          "1001"
drivers/network/dd/netkvm/wxp/netkvm.inx	
  57 HKR, Ndi\Params\ConnectRate,        Default,    0,          "1001"
```